00-open-a-shapefile.Rmd

```diff
@@ -159,31 +155,26 @@ aoiBoundary <- readOGR("boundaryFiles/HARV", "HarClip_UTMZ18")
 ```
 
 After loading a new data set we also want to view the attributes of the layer.
-This can be done simple by entering the file name.
+This can be done simple by entering the file name. Alternatively, we can use the `attributes()` function.
```

Comment:
Okay.  I'll start adding the line for all notes (further explanation of what is discussed) and bonus (extra materials that go beyond what is discussed).  I think we reserve box notices for anything that really needs to stand out -- there should be few enough of these that they could be an addition during the final reviewing process or even after testing out the lessons here at NEON.